srcpkgs/rust/template

```diff
@@ -127,6 +127,7 @@ do_configure() {
 		incremental = false
 		parallel-compiler = false
 		channel = "stable"
+		description = "Void"
```

Comment:
Causes this suffix in the compiler version output:
```
$ rustc --version
rustc 1.63.0 (Void)
```

Reply:
hm maybe `Void Linux`? (a bit of a nitpick)

Reply:
We can do that for the next version, but I'd prefer not rebuilding rust another 7 times to get the new distfiles into place with that change included :D

Reply:
it's fine for the bootstrap distfiles to not have that minor difference I think, I'll make the change when merging.